src/app/helptext/apps/apps.ts

```diff
@@ -561,7 +561,7 @@ export default {
     title: T('Warning'),
     message: T(`iXsystems does not audit or otherwise validate the contents of third-party applications catalogs. \
     It is incumbent on the user to verify that the new catalog is from a trusted source and that the third-party \
-    properly audits its chart contents. Failure to exercise due diligence can expose the user and their data to \
+    properly audits its chart contents. Failure to exercise due diligence may expose the user and their data to \
     the some or all of the following:<br/>\
```

Comment:
```suggestion
    some or all of the following:<br/>\
```
I'm not an native English speaker, but it doesn't sounds correct to me.
If that's correct, please ignore this.